src/test/kotlin/org/nosphere/apache/rat/AbstractPluginTest.kt

```diff
@@ -41,11 +41,11 @@ abstract class AbstractPluginTest(
         @Parameterized.Parameters(name = "{0}")
         @JvmStatic
         fun testMatrix() = listOf(
-            TestMatrix(GradleVersion.version("8.0-rc-1"), true),
-            TestMatrix(GradleVersion.version("7.6"), true),
-            TestMatrix(GradleVersion.version("7.0"), true),
+            TestMatrix(GradleVersion.version("8.0.2"), true),
+            TestMatrix(GradleVersion.version("7.6.1"), true),
+            TestMatrix(GradleVersion.version("7.0.2"), true),
             TestMatrix(GradleVersion.version("6.9.2"), true),
-            TestMatrix(GradleVersion.version("6.0"))
+            TestMatrix(GradleVersion.version("6.0.1"))
```

Comment:
Let's stick with first and last minor of versions here.
We should test 6.0 instead of 6.0.1, 7.0 instead of 7.0.2, 8.0 and 8.0.2 etc...

Reply:
b0fd57aed11490a71a7839f7abcd3b9fc1ffee34